Лаб 1/1.py

```diff
@@ -0,0 +1,19 @@
+numbers = [2, -93, -2, 8, None, -44, -1, -85, -14, 90, -22, -90, -100, -8, 38, -92, -45, 67, 53, 25]
+
+# TODO заменить значение пропущенного элемента средним арифметическим
+
+numbers1 = numbers[5:] # после неизвестного
+numbers2 = numbers[:4] # до неизвестного
+n = numbers2 + numbers1
```

Comment:
Решение не плохое, но в целях оптимизации кода можно предусмотреть суммирование сразу в рамках одно переменной 
```suggestion
numbers_int = numbers[:4] + numbers[5:]
```
А если учитывать что мы до этого ищем индекс автоматически, то эта строка примет следующий вид:
```suggestion
numbers_int = numbers[:none_index] + numbers[none_index + 1:]
```
Во втором случае мы используем одну переменную под индекс искомого значения, но при выделении части списка после `None`, нам надо не забыть увеличить индекс на 1

Ну и если еще больше это оптимизировать, то мы можем вместо переменной индекса сразу ставить функцию поиска индекса:
```suggestion
numbers_int = numbers[:numbers.index(None)] + numbers[numbers.index(None) + 1:]
```